apps/passport-server/src/services/generic-issuance/pipelines/PretixPipeline.ts

```diff
@@ -391,6 +418,12 @@ export class PretixPipeline implements BasePipeline {
     );
   }
 
+  private async getAllManualTickets(): Promise<ManualTicket[]> {
+    return (this.definition.options.manualTickets ?? [])
+      .concat(await this.manualTicketDB.loadAll(this.id))
+      .filter((t) => !!t);
```

Comment:
To me this suggests that there's something wrong with the types of the two arrays being concatenated. They either contain manual tickets or nothing, but should never contain undefined or null objects. I'm curious as to why this filter was necessary.

Reply:
This is leftover code that I've now deleted.